packages/webapp/src/Routes.jsx

```diff
@@ -603,7 +600,8 @@ const Routes = () => {
             <Route path="/estimated_revenue" exact component={EstimatedRevenue} />
             <Route path="/labour" exact component={Labour} />
             <Route path="/other_expense" exact component={OtherExpense} />
-            <Route path="/expense_detail" exact component={ExpenseDetail} />
+            <Route path="/expense/:expense_id/" exact component={ExpenseDetail} />
```

Comment:
Is the trailing backslash needed?

Reply:
No, it's not! Thanks! It's a holdover from when the endpoint was `/:expense_id/details`. I'll make a note to fix that in a subsequent PR (router accepts it works but it's technically incorrect).